docs/source/working-notes/troyraen/Rubin-stream-test/README.md

```diff
@@ -0,0 +1,309 @@
+# Connect Pitt-Google to the Rubin alert stream testing deployment
+
+THIS README HAS BEEN MOVED TO THE DIRECTORY broker/consumer/rubin.
```

Comment:
Maybe something like: the active copy of this README.md is in broker/consumer/rubin and will be kept up to date as the main LSST alert stream developments.  This README.md here is specifically for this test stream.  They are currently the same, but in the future will diverge.